libcaf_io/caf/io/basp/remote_message_handler.hpp

```diff
@@ -103,6 +103,8 @@ class remote_message_handler {
     auto t0 = telemetry::timer::clock_type::now();
     if (!source.apply(msg)) {
       CAF_LOG_ERROR("failed to read message content:" << source.get_error());
+      detail::sync_request_bouncer srb{sec::malformed_message};
+      srb(src, mid);
```

Comment:
This would currently only send this error message when using requests. I think failing to deserialize a message is severe enough that we should always send the error.

```suggestion
      src->enqueue(nullptr, mid.response_id(),
                   make_message(make_error(sec::malformed_message)),
                   nullptr);
```